systemtest/README.md

```diff
@@ -0,0 +1,17 @@
+# System Test

```

Comment:
Can you change the heading to AdoptOpenJDK System Tests ?

Reply:
Done